modules/images/webp-uploads/load.php

```diff
@@ -285,3 +288,47 @@ function webp_uploads_remove_sources_files( $attachment_id ) {
 }
 
 add_action( 'delete_attachment', 'webp_uploads_remove_sources_files', 10, 1 );
+
+/**
+ * Filter on `wp_get_missing_image_subsizes` acting as an action for the logic of the plugin
+ * to determine if additional mime types still need to be created.
+ *
+ * @since n.e.x.t
+ *
+ * @see   wp_get_missing_image_subsizes()
+ *
+ * @param array[] $missing_sizes Associative array of arrays of image sub-size.
+ * @param array   $image_meta    The metadata from the image.
+ * @param int     $attachment_id The ID of the attachment.
+ *
+ * @return array[] $missing_sizes Associative array of arrays of image sub-size.
```

Comment:
Return docs shouldn't have any variable name, and there should be no blank line in between `@param` and `@return` per WP docs standards. Also, `array[]` isn't a valid type, WP core tends to always use just `array`. If anything, it could be `[]array` which would be the correct version for "array of arrays".
```suggestion
 * @param array   $missing_sizes Associative array of arrays of image sub-sizes.
 * @param array   $image_meta    The metadata from the image.
 * @param int     $attachment_id The ID of the attachment.
 * @return array Associative array of arrays of image sub-sizes.
```